renovate.json

```diff
@@ -3,15 +3,15 @@
     "extends": [
       "config:base"
     ],
-    "baseBranches": ["main", "release-2.11", "release-2.10"],
+    "baseBranches": ["main", "release-2.12", "release-2.11"],
```

Comment:
Does renovate look at this file on the branch?

Reply:
I am not sure. But it is better to keep this one up-to-date too, at least for completeness.

Reply:
I disagree. I find it confusing to make unnecessary changes. If we want to include new branch, we should do it in file that's actually used.

Furthermore if we merge this branch into `main` after creating rc-0, we will effectively stop getting updates for release-2.10, even though release-2.12 wasn't released yet.